CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# These owners are the maintainers and approvers of this repo
+*       @dapr/maintainers-go-sdk @dapr/approvers-go-sdk @dapr/maintainers-dapr @dapr/approvers-dapr
```

Comment:
I don't think we should make go-sdk maintainers code owners here because this repo has the server side implementation which is in runtime.